src/utilities.jl

```diff
@@ -174,6 +174,45 @@ function parsedocs(mod::Module)
     end
 end
 
+"""
+$(:SIGNATURES)
+
+Decides whether a length of method is too big to be visually appealing.
+"""
+method_length_over_limit(len::Int) = len > 60
+
+function printmethod_format(buffer::IOBuffer, binding::String, args::Vector{String}, kws::Vector{String}; return_type = "")
+
+    sep_delim = " "
+    paren_delim = ""
+    indent = ""
+
+    if method_length_over_limit(
+            length(binding) +
+            1 +
+            sum(length.(args)) +
+            sum(length.(kws)) +
```

Comment:
Same again:

```suggestion
           sum(length, kws) +
```

Reply:
oh wow, this actually crashes with empty vectors :cry: 

```
julia> sum(length, [])
ERROR: ArgumentError: reducing over an empty collection is not allowed
```

...that's weird, will ask on slack.

Reply:
`sum(length, []; init = 0)` instead?

Reply:
True, already fixed :]

Reply:
Apparently tis isn't easily possible with Julia <1.6. Rolling back.

:cry: tho.